autogen/agentchat/chat.py

```diff
@@ -0,0 +1,15 @@
+import logging
```

Comment:
minor comment
Should this live in a datamodel.py class?
At some point, it might be good to make most classes and their args/return values based on datamodels.

Reply:
Noted! Let's discuss this potential change in a future PR.